src/card.css

```diff
@@ -0,0 +1,93 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  width: 200px;
+
+  background: var(--bg-color);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
```

Comment:
Probably it's not a good idea to hardcode height, I think just use width would be great

Reply:
it's didn't pass tests without height

Reply:
Sorry, you're right. I had the same problem)